spring-amqp/src/main/java/org/springframework/amqp/core/MessageProperties.java

```diff
@@ -434,6 +434,7 @@ public void setConsumerQueue(String consumerQueue) {
 	 * The x-delay header (outbound).
 	 * @return the delay.
 	 * @since 1.6
+	 * @deprecated in favor of {@link #getDelayLong()}
```

Comment:
This method probably also can delegate to the `getDelayLong`.

You also have missed to add `@Deprecated(since = "3.1.2", forRemoval = true)` for both integer-based methods.